src/keywords.cpp

```diff
@@ -152,6 +153,7 @@ static const chunk_tag_t keywords[] =
    { "else",             CT_PP_ELSE,       LANG_ALL | FLAG_PP                                                          }, // PAWN
    { "elseif",           CT_PP_ELSE,       LANG_PAWN | FLAG_PP                                                         }, // PAWN
    { "emit",             CT_PP_EMIT,       LANG_PAWN | FLAG_PP                                                         }, // PAWN
+   { "endasm",           CT_PP_ENDASM,     LANG_ALL | FLAG_PP                                                          },
```

Comment:
No indeed, the test was not necessary, and so neither is this keyword. I guess it will become necessary if we can switch to ASM indent in these section.